pkg/controllers/user/alert/watcher/pod.go

```diff
@@ -172,17 +173,23 @@ func (w *PodWatcher) checkPodRestarts(pod *corev1.Pod, alert *v3.ProjectAlertRul
 				if err != nil {
 					logrus.Warnf("Failed to get cluster for %s: %v", w.clusterName, err)
 				} else {
-					clusterDisplayName = cluster.Spec.DisplayName
+					clusterDisplayName = common.GetClusterAlertDisplayName(cluster.Spec.DisplayName, w.clusterName)
```

Comment:
I feel like code on line 176-183 repeats in 277-284. We should extract it to the common method, and use it

Reply:
updated